server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/StartAuditRound.java

```diff
@@ -256,7 +256,7 @@ public List<ContestResult> countAndSaveContests(final Set<ContestToAudit> cta) {
     LOGGER.debug(String.format("[countAndSaveContests: cta=%s]", cta));
     final Map<String, AuditReason> tcr = targetedContestReasons(cta);
 
```

Comment:
I think the addition of the flag (useManifests = true) deserves a little comment.